File: 1302-delete-characters-to-make-fancy-string/1302-delete-characters-to-make-fancy-string.java
class Solution {
    public String makeFancyString(String s) {
        StringBuilder sb =new StringBuilder();
        sb.append(s.charAt(0));
        int c=1;
        for(int i=1;i<s.length();i++)
        {
            if(s.charAt(i)==sb.charAt(sb.length()-1)){
                c++;
                if(c<3)
                sb.append(s.charAt(i));
            }
            else{
                c=1;
                sb.append(s.charAt(i));
            }

        }
        return sb.toString();
    }
}

File: 1302-delete-characters-to-make-fancy-string/1302-delete-characters-to-make-fancy-string.py
class Solution:
    def makeFancyString(self, s: str) -> str:
        res=""
        res=s[0]
        c=1
        for i in range(1,len(s)):
            if s[i]==s[i-1]:
                c+=1
            else:
                c=1
            if c<3:
                res=res+s[i]
        return res